bgpd/rfapi/rfapi_import.h

```diff
@@ -94,7 +94,7 @@ extern void rfapiImportTableRefDelByIt(struct bgp *bgp,
  * Construct an rfapi nexthop list based on the routes attached to
  * the specified node.
  *
- * If there are any routes that do NOT have BGP_INFO_REMOVED set,
+ * If there are any routes that do NOT have BGP_PATH_REMOVED set,
  * return those only. If there are ONLY routes with BGP_INFO_REMOVED,
```

Comment:
Should be BGP_PATH_REMOVED :)

Reply:
I had issue in adding the comment at the right place, but this is still applicable. On the new change, rfapi_import.h file still has this.